classes/ActionScheduler_ListTable.php

```diff
@@ -327,8 +339,10 @@ public function display_admin_notices() {
 			// No lock set or lock expired
 			if ( false === $async_request_lock_expiration || $async_request_lock_expiration < time() ) {
 				$in_progress_url       = add_query_arg( 'status', 'in-progress', remove_query_arg( 'status' ) );
-				$async_request_message = sprintf( __( 'A new queue has begun processing. %sView actions in-progress%s', 'action-scheduler' ), '<a href="' . esc_url( $in_progress_url ) . '">', ' &raquo;</a>' );
+				/* translators: %s: process URL */
+				$async_request_message = sprintf( __( 'A new queue has begun processing. <a href="%s">View actions in-progress &raquo;</a>', 'action-scheduler' ), esc_url( $in_progress_url ) );
```

Comment:
Some HTML tags are better to just keep in the translatable string, since it's very easy to miss or incorrect translate `%sView` or `s%s`.